editor/src/clj/dev.clj

```diff
@@ -44,7 +44,7 @@
   (ffirst (g/targets-of (workspace) :resource-map)))
 
 (defn app-view []
-  (ffirst (g/targets-of (project) :_node-id)))
+  (ffirst (g/targets-of (project) :selected-node-ids-by-resource-node)))
```

Comment:
We changed the connection order at some point in the past, causing this to return `nil`. But now it works!